src/metabase/lib/core.cljc

```diff
@@ -492,7 +493,12 @@
   unique-name-generator
   unique-name-generator-with-options]
  [lib.validate
-  find-bad-refs]
+  duplicate-column
+  find-bad-refs
+  missing-column
+  missing-table-alias
+  syntax-error
+  validation-error]
```

Comment:
We have to export these with better names... `lib/duplicate-column` to construct an **error** map is super duper mega confusing. 

Look at the impl for `metabase.util.namespaces/import-fns`, you can rename stuff as you export it.